src/utils/useForceUpdate.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export default function useForceUpdate() {
```

Comment:
I wouldn’t call it use**Force**Update, forceUpdate triggers re-render synchronously, which is not possible with hooks. Maybe just call it useUpdate

Reply:
I'll change it